src/Timeline/styled.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'reakit/styled';
+import { palette } from 'styled-tools';
+import Box from 'reakit/Box';
+
+export const Row = styled(Box)`
+  display: flex;
+  position: relative;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &::before {
+    content: '';
+    border: 1px solid ${props => props.lineColor || palette('textTint')};
+    position: absolute;
+    height: 100%;
+    left: 0.25rem;
+    margin-top: 1.25rem;
+  }
+
+  &:last-child::before {
+    border: 0;
+  }
+`;
+
+export const StepBullet = styled.span`
+  background-color: ${props => props.color || palette('textTint')};
+  border-radius: 50%;
+  display: inline-flex;
+  height: 0.625rem;
```

Comment:
uhmm, in order to make the Bullet `em` as suggested, I would need to make the "Step" content a parent div of Bullet, which is not case and would involve a whole refactoring of the current structure, Also, `em` can be tricky to use with nested elements. so I think using `rem` is much safer in this particular case and easier as it will always remain relative to the root element.

```html
<div class="Row">
  <span class="Bullet"/>
  <div class="Step">...</div>
</div>
 ...
```

Reply:
fair enough!